dev-docs/logging.md

```diff
@@ -130,7 +130,7 @@ Use `with_subscriber` to attach a subscriber to an async block.
 ```rust
     #[tokio::test]
     async fn test_async() {
-        async{...}.with_subscriber(assert_snapshot_subscriber!())
+        async{...}.with_subscriber(assert_snapshot_subscriber!()).await
```

Comment:
Despite my need to make minor tweaks I really appreciated both of these dev docs!